src/main/resources/page/js/dummy-page-loader.js

```diff
@@ -0,0 +1,11 @@
+class DummyPageLoader {
+    constructor() {}
+
+    static async load() {
+        let dateOrTime = await DummyRESTAPICaller.getData('http://localhost:8080/sekta/api/dummy');
```

Comment:
Should we define the URL as constant and could we move it out to the DummyRESTAPICaller class?
It belongs there, now we have a DummyRESTAPICaller class, but this class does not know what is the URL (This is the only class, who should know this detail -> Encapsulation -> OOP principle)